src/main/resources/steps/composite.steps

```diff
@@ -0,0 +1,19 @@
+Composite: When I login with correct user
```

Comment:
I would suggest you to create a composite like

When I login with username \`$user\` and password \`$password\`

It will let you use this composite step in both positive and negative scenarios by applying different values.